### ISSUE TITLE
Moon output hard to parse, could the 'first quarter'/'full moon' text be moved to the top? "Quiet" moon output?

### ISSUE THREAD

**Issue by mikedesu** [chubin/wttr.in#701]
Hi. 

I'd like to be able to get a 'quiet' version of the moon output, perhaps without displaying the ASCII moon image. This is difficult to grep in the current output when I curl `wttr.in/moon`. If the text were moved to the top of the moon image, I could grep it out myself, but at this point it would probably be easy to just add a `q` and/or `Q` option for the moon phase. 

Thank you for everything you've done! Keep on rocking \m/

**Comment by chubin** [comment]
@mikedesu hi! It is a great feature. So you would like just to have moon phase name in text form?

We have this:

```
~$ curl wttr.in/?format=%m; echo
🌗
~$ curl wttr.in/?format=%M; echo
22
```

but probably you are right, and it would be great to have the same in text form.
